### PR TITLE
Header component: creation, styling and adjusting

### PR DESCRIPTION
**Pull request by TEZv**
Added styles, and **BEM** class names, made **dynamic** measurements, adjusted **image names**, 
and added **dark styles** to styles.css that will be **adjusted further**.

Please, check and comment for improvements or pull to **dev** branch ☺

### CODE REVIEW

File: lifechanger/src/app/header/header.component.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  z-index: 100;
+  & .header__nav {
+    display: flex;
+    justify-content: space-between;
+    position: fixed;
+    background-color: var(--white);
+    top: 2.6em;
+    left: 20.8em;
+    width: 76rem;
+    /* height: 59px; */
```
Comment (ruslandzoyb) [inline lifechanger/src/app/header/header.component.css]: Is it required comment ? If we talking about comments in code. It is good approach to use for documentation of some logic (example xml comments in C# - attached image) Any other comments that related to code is better not to left. From my personal experience at project, we don't use comments for hiding some part of logic. So, my suggestion - is to remove them, if you don't mind. But i open for discussion, if you have some points here 
image

File: lifechanger/src/app/header/header.component.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      justify-content: center;
+      align-items: center;
+      width: 52em;
+      margin-right: 12px;
+      & .header__nav-links {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        width: 40em;
+        /* height: 26px; */
```
Comment (ruslandzoyb) [inline lifechanger/src/app/header/header.component.css]: same here

File: lifechanger/src/styles.css [hunk 3]
```diff
@@ -1,4 +1,9 @@
 /* You can add global styles to this file, and also import other style files */
+/* -------------- `14-18.4` Change the colour names,
```
Comment (ruslandzoyb) [inline lifechanger/src/styles.css:2]: I think this type of comment is actually fine, we can leave it

File: lifechanger/src/styles.css [hunk 4]
```diff
@@ -41,4 +51,159 @@ a {
...
+  transition: 0.4s;
+  width: 26px;
+}
+.slider.round {
+  border-radius: 34px;
+}
+.slider.round:before {
+  border-radius: 50%;
+}
+/* @media screen and (max-width: 479px) {
```
Comment (ruslandzoyb) [inline lifechanger/src/styles.css]: Same question

File: lifechanger/src/app/header/header.component.css (hunk 1 above)
Comment (TEZv) [reply lifechanger/src/app/header/header.component.css]: Thank you, dear Mentor.
I deleted the unnecessary comments, please, check once again:)

File: lifechanger/src/app/header/header.component.css (hunk 2 above)
Comment (TEZv) [reply lifechanger/src/app/header/header.component.css]: Thank you, dear Mentor.
I deleted the unnecessary comments, please, check once again:)

File: lifechanger/src/styles.css (hunk 3 above)
Comment (TEZv) [reply lifechanger/src/styles.css:2]: Thank you, Mentor.
I added it here to not forget about further enhancement and implementation.

File: lifechanger/src/styles.css (hunk 4 above)
Comment (TEZv) [reply lifechanger/src/styles.css]: Thank you, dear Mentor.
I deleted the unnecessary comments, please, check once again:)

**Pull request merged**